api/api/templates/admin/api/media_report/change_form.html

```diff
@@ -87,7 +83,11 @@ <h2>Media details</h2>
       <div class="flex-container">
         <label>Tags</label>
         <div class="readonly">
-          <p>{{ tags }}</div>
+          {% for provider, provider_tags in tags.items %}
+            <p><em>{{ provider }}</em></p>
+            <p> {% for tag in provider_tags %}{{ tag }}, {% endfor %}</p>
```

Comment:
Nit: This will also introduce a trailing comma on the last item. I think we could use `forloop.last` as the check for this? 

![image](https://github.com/WordPress/openverse/assets/10214785/9d99c65a-1a26-4e4f-98dd-d6ddb4cc4e3d)

Reply:
It should be done with the join filter: https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#std-templatefilter-join

```suggestion
            <p>{{ provider_tags | join:', ' }}</p>
```